includes/emails/class-emails.php

```diff
@@ -488,6 +488,19 @@ public static function get_email_config_by_type( $type ) {
 			/** Only attempt to create the email post if wp-includes/pluggable.php is loaded. */
 			return false;
 		} else {
+			// Make sure newsletters color palette is updated with latest theme colors.
+			if ( self::supports_emails() ) {
```

Comment:
Despite this check, the [test are failing](https://app.circleci.com/pipelines/github/Automattic/newspack-plugin/9667/workflows/2352b9aa-477b-420f-82fc-410773375045/jobs/46291).

Reply:
Ah. I think this is because the `Plugin_Manager` downloads the latest release for newsletters, which does not yet have the required method: https://github.com/Automattic/newspack-plugin/blob/725786b70e43ff8d8b9e3789c43cea13f70d4eb1/includes/class-plugin-manager.php#L144

I guess we can hold off on moving forward with this one until the next newsletters release.